torch/nn/_VF.py

```diff
@@ -0,0 +1,14 @@
+import torch
+import sys
+import types
+
+
+class VFModule(types.ModuleType):
```

Comment:
I think this is a common pattern now where there is a module that wraps an object. Can this be turned into a utility class and used in both places?

Reply:
Will do in a followup PR